src/ui/src/builder/journal/BuilderJournalSearchbar.vue

```diff
@@ -33,32 +41,29 @@
 import WdsSelect from "@/wds/WdsSelect.vue";
 import WdsTextInput from "@/wds/WdsTextInput.vue";
 import WdsButton from "@/wds/WdsButton.vue";
-import { computed, PropType } from "vue";
-
-const statusOptions = [
-	{ value: "success", label: "Success" },
-	{ value: "error", label: "Error" },
-	{ value: "stopped", label: "Stopped" },
-];
-const triggerOptions = [
-	{ value: "On demand", label: "On demand" },
-	{ value: "UI", label: "UI" },
-	{ value: "API", label: "API" },
-	{ value: "Cron", label: "Scheduled" },
-];
+import { computed } from "vue";
+import {
+	STATUS_OPTIONS,
+	TRIGGER_OPTIONS,
+	INSTANCE_TYPE_OPTIONS,
+} from "./journalConstants";
+import type { JournalFilters } from "./journalTypes";
 
-const searchText = defineModel("search", { type: String });
-const statuses = defineModel("statuses", { type: Array as PropType<string[]> });
-const triggers = defineModel("triggers", { type: Array as PropType<string[]> });
+const filters = defineModel<JournalFilters>("filters", { required: true });
```

Comment:
It's better to rely on runtime check

```suggestion
const filters = defineModel("filters", { type: Object as PropType<JournalFilters>, required: true });
```

Reply:
In runtime, does it check the object's shape and props, or just type check if it is an object?